transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollDomainSocketChannel.java

```diff
@@ -37,6 +37,13 @@ public EpollDomainSocketChannel(Channel parent, FileDescriptor fd) {
         super(parent, fd.intValue());
     }
 
+    /**
+     * Creates a new {@link EpollDomainSocketChannel} from an existing {@link FileDescriptor}
+     */
+    public EpollDomainSocketChannel(FileDescriptor fd) {
```

Comment:
Why don't we just accept an `int` instead of `FileDescriptor`?  `FileDescriptor` can be any transport implementation's `FileDescriptor` implementation, so it doesn't prevent a user from passing non-epoll `FileDescriptor`, which can be seen confusing.